src/domain/opportunity/opportunity.entity.ts

```diff
@@ -100,6 +101,17 @@ export class Opportunity extends BaseEntity
   )
   aspects?: Aspect[];
 
+  @Field(() => [Relation], {
+    nullable: true,
+    description: 'The set of relations for this Opportunity',
+  })
+  @OneToMany(
+    () => Relation,
+    relation => relation.opportunity,
+    { eager: true, cascade: true }
```

Comment:
Again eager in one-to-many?

Reply:
Yes but this is at the end of a containment tree i.e. relations are simple entities that are contained by opportunities - so yes think this is the best option...

Reply:
and as before, first lets see if this actually causes performance issues and then see where to break the dependency chain for eager loading...

Reply:
I am approving it but even if it is at the end of a long-ish tree, all eager relations are loaded and it will impact all the items up to the top.